wren-ui/src/components/pages/setup/dataSources/MySQLProperties.tsx

```diff
@@ -73,7 +73,7 @@ export default function MySQLProperties(props: Props) {
           },
         ]}
       >
-        <Input.Password placeholder="input password" />
+        <Input.Password placeholder="************" />
```

Comment:
The `<MySQLProperties/>` component is used in the onboarding process and the data source settings. 
This change will be different from what is expected on the onboarding process side, we still want to keep the placeholder as the **input password** string.